documentation/docs/30-advanced/25-errors.md

```diff
@@ -87,6 +87,8 @@ const Sentry: any;
 
 // @filename: index.js
 // ---cut---
+import * as Sentry from '@sentry/svelte';
```

Comment:
Since this is example is for the server, we need to import `@sentry/node` here as well and also initialize the SDK
```suggestion
import * as Sentry from '@sentry/node';

Sentry.init({...})
```